corehq/apps/domain/decorators.py

```diff
@@ -80,51 +80,52 @@ def login_and_domain_required(view_func):
     def _inner(req, domain, *args, **kwargs):
         user = req.user
         domain_name, domain = load_domain(req, domain)
-        if domain:
-            if user.is_authenticated and user.is_active:
-                if not domain.is_active:
-                    msg = _((
-                        'The domain "{domain}" has not yet been activated. '
-                        'Please report an issue if you think this is a mistake.'
-                    ).format(domain=domain_name))
-                    messages.info(req, msg)
-                    return HttpResponseRedirect(reverse("domain_select"))
-                couch_user = _ensure_request_couch_user(req)
-                if couch_user.is_member_of(domain):
-                    if domain.two_factor_auth and not user.is_verified() and not couch_user.two_factor_disabled:
-                        return TemplateResponse(
-                            request=req,
-                            template='two_factor/core/otp_required.html',
-                            status=403,
-                        )
-                    else:
-                        return view_func(req, domain_name, *args, **kwargs)
+        if not domain:
+            msg = _(('The domain "{domain}" was not found.').format(domain=domain_name))
```

Comment:
nit: in a user facing message I think the term for a 'domain' is a 'project'. Same for other places.

Reply:
I actually didn't add this, just removed one level of indentation here. I'm going to leave this as is for now.